.cirrus.yml

```diff
@@ -70,7 +70,7 @@ task:
   env:
     HOME: /root
     CIRRUS_WORKING_DIR: /home/runc
-    GO_VERSION: "1.19"
+    GO_VERSION: "1.19.8"
```

Comment:
should we change the others to use and explicitly .patch versions as well? That way we know they're all on the same version (and less risk of one being left behind).

It's a bit more maintenance though, but perhaps we can have a script or job to update 🤔

Reply:
No we should not. This is the only place which interprets the version number literally (i.e. 1.9 means 1.9, not 1.9.x like in other places).

If we start specifying minor releases everywhere (yes we can do that), we'll have to bump them every time. I'd like to avoid that.

I would love to avoid that here as well, but I don't know a way to do that (without making the whole thing much more complicated than it is).

Reply:
Now, there is a problem that Go 1.20 means Go 1.20.x in some places (Dockerfile, GHA jobs) and Go 1.20 (the initial 1.20 release) in some other places.

Apparently this is going to be fixed with Go 1.21 (they will have it as Go 1.21.0), see https://github.com/golang/go/issues/57631.

The next problem is, how to say "I want to download the latest Go 1.20.x" without explicitly specifying x.

Reply:
OK, I managed to write some logic to download the latest 1.20.x given 1.20 as an input, so we don't have to specify the minor release explicitly; PTAL.